config/environments/production.rb

```diff
@@ -22,8 +22,10 @@
   config.action_mailer.default_options = { from: Figaro.env.email_from }
   config.action_mailer.delivery_method = if Figaro.env.disable_email_sending == 'true'
                                            :test
+                                         elsif Figaro.env.mandrill_api_token.present?
+                                           :mandril
```

Comment:
typo? is it missing a second L? `:mandrill` ?

Reply:
Good catch. Fixed it in a follow up commit